Data-Pro-Test/Scripts/email-form.js

```diff
@@ -0,0 +1,44 @@
+﻿
```

Comment:
what is this file. You have already - Data-Pro-Test/Scripts/app/email-form.js file